### ISSUE TITLE
Dolphin Crashes on Android 13

### ISSUE THREAD

**Issue by hunter1026** [libretro/RetroArch#15531]
as soon as i launch a wii iso it goes black screen and crashes every time. the system files are present.

**Comment by hizzlekizzle** [comment]
yes, this is a known thing. I believe it started with the update to Android v9.0 Pie. At least, that's when it started on my Shield ATV when it had previously be working fine (just did a Shield Experience update; no change to RetroArch/core).

**Comment by hunter1026** [comment]
i wonder if its a retroarch issue or a core issue?

**Comment by hizzlekizzle** [comment]
Dunno. From our perspective, it's a Google/Android issue, since a thing that was working suddenly stopped when they updated things. I know that doesn't get you (or us) anywhere, though.

**Comment by lim2222** [comment]
I grap an old backup core from my old phone it works with no crash on android 10 with fastmen off.Drop that core here in case anybody want.
[dolphinNext_libretro.zip](https://github.com/libretro/RetroArch/files/12208777/dolphinNext_libretro.zip)
